src/lib/workers/workerUtils.js

```diff
@@ -6,13 +6,16 @@
 var { parentPort } = require('worker_threads');
 
 module.exports.workerTaskProcessor = function (taskProcessor) {
-    parentPort.on('message', (msg) => {
-        taskProcessor(msg.msg)
-            .then(data => { 
-                msg.channelWorkerPort.postMessage(data);
-            }).catch(err => {
-                msg.channelWorkerPort.postMessage((err instanceof Error) ? err.message : err);
-            });
-    });
+    if (parentPort){
```

Comment:
do we need handle else case here?

Reply:
Good question~  @deepak-ms   Can you help us take a look at it? I'm not sure if I need to handle else case here, and just keep the old behavior.

Reply:
I don't use the workerPool when packing this lib now. It seems that we don't need to handle else and when the parentPort is null, we just keep the old behavior and don't run the taskProcessor.